src/net/RTP/RTPChannel.cs

```diff
@@ -387,7 +387,7 @@ public void Close(string reason)
         /// <param name="buffer">The data to send.</param>
         /// <returns>The result of initiating the send. This result does not reflect anything about
         /// whether the remote party received the packet or not.</returns>
-        internal virtual SocketError Send(RTPChannelSocketsEnum sendOn, IPEndPoint dstEndPoint, byte[] buffer)
+        public virtual SocketError Send(RTPChannelSocketsEnum sendOn, IPEndPoint dstEndPoint, byte[] buffer)
         {

```

Comment:
Yes, if you use RTPSession, there are no such problems. But our business does not need such rich functions of RTPSession first, such as sending H264 data frames, etc., we need to implement a LiteRTPSession, which only contains the data receiving function, and these basic functions need to rely on RTCPSession and RTPChannel, at this time We will find that the LiteRTPSession we implemented by ourselves cannot access all of the above methods, which will block the implementation.

Reply:
In addition, currently RTPSession only supports H264 and VP8 format RTP Payload, and our RTP Payload is MPEG-PS, so instead of modifying RTPSession to add these functions, it should be re-implemented, because this is business-specific and not a standard.